ui/src/scss/_patterns_buttons.scss

```diff
@@ -65,4 +65,13 @@
   .p-button--link:not(:last-of-type):not(:only-of-type) {
     margin-right: 0;
   }
+
+  button.is-thin {
```

Comment:
Is this a case that needs to make its way into Vanilla?

Reply:
I think so. We used thin buttons in RBAC, and we have some slightly different thin buttons used in action table cells in MAAS.